pytradfri/device.py

```diff
@@ -189,8 +188,9 @@ def set_dimmer(self, dimmer, *, index=0, transition_time=None):
         dimmer: Integer between 0..254
         transition_time: Integer representing tenth of a second (default None)
         """
-        if dimmer < 0 or dimmer > 254:
-            raise ValueError('Dimmer value must be between 0 and 254.')
+        if dimmer < RANGE_BRIGHTNESS[0] or dimmer > RANGE_BRIGHTNESS[1]:
```

Comment:
Only dimmer values are actively validated. Should we use this opportunity to validate the rest too?